_inc/lib/admin-pages/class.jetpack-about-page.php

```diff
@@ -0,0 +1,557 @@
+						</p>
+						<p>
+							<a href="https://automattic.com/jobs" target="_blank"  class="jptracks" data-jptracks-name="jetpack_about_work_with_us">
+								<?php esc_html_e( 'Come work with us', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div>
+				</div>
+			</div>
+
+			<div class="jetpack-about__colophon">
+				<h3><?php esc_html_e( 'Popular WordPress services by Automattic', 'jetpack' ); ?></h3>
+				<ul class="jetpack-about__services">
+				<?php $this->display_plugins(); ?>
+				</ul>
+
+				<p class="jetpack-about__services-more"><?php echo wp_kses( __( 'For even more of our WordPress plugins, please <a href="https://profiles.wordpress.org/automattic/#content-plugins" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_wporg_profile">take a look at our WordPress.org profile</a>.', 'jetpack' ), array( 'a' => array( 'href' => array(), 'target' => array(), 'class' => array(), 'data-jptracks-name' => array() ) ) ); ?></p>
+			</div>
+		</div>
+		<?php
+	}
+
+	function display_plugins() {
+		$plugins_allowedtags = array(
+			'a'       => array(
+				'href'   => array(),
+				'title'  => array(),
+				'target' => array(),
+			),
+			'abbr'    => array( 'title' => array() ),
+			'acronym' => array( 'title' => array() ),
+			'code'    => array(),
+			'pre'     => array(),
+			'em'      => array(),
+			'strong'  => array(),
+			'ul'      => array(),
+			'ol'      => array(),
+			'li'      => array(),
+			'p'       => array(),
+			'br'      => array(),
+		);
+
+		// slugs for plugins we want to display
+		$a8c_plugins = array(
+			'akismet',
+			'wp-super-cache',
+			'vaultpress',
+			'polldaddy',
+		);
+
+		// need this to access the plugins_api() function
+		include_once( ABSPATH . 'wp-admin/includes/plugin-install.php' );
+
+		foreach ( $a8c_plugins as $slug ){
+			$args = array(
+				'slug'	=> $slug,
+				'fields'	=> array(
+					'added'						=> false,
+					'author'						=> false,
+					'author_profile'				=> false,
+					'banners'					=> false,
+					'contributors'				=> false,
+					'donate_link'				=> false,
+					'homepage'					=> false,
+					'reviews'					=> false,
+					'screenshots'				=> false,
+					'support_threads'			=> false,
+					'support_threads_resolved'	=> false,
+					'sections'					=> false,
+					'tags'						=> false,
+					'versions'					=> false,
+
+					'compatibility'				=> true,
+					'downloaded'					=> true,
+					'downloadlink'				=> true,
+					'icons'						=> true,
+					'last_updated'				=> true,
+					'num_ratings'				=> true,
+					'rating'						=> true,
+					'requires'					=> true,
+					'requires_php'				=> true,
+					'short_description'			=> true,
+					'tested'						=> true,
+				),
+			);
+
+			// should probably add some error checking here too
+			$api = plugins_api( 'plugin_information', $args );
+			$plugins[] = $api;
```

Comment:
Can we also custom sort this list, or randomize it or something? It felt really weird when I viewed this to see Crowdsignal (low rating, no icon) show up, vs something like WooCommerce (super popular, well-branded, we'd like to encourage it being installed, etc).

Reply:
Urrr oops, I guess this applies actually to the list of plugins we load up above, on line 207.

Reply:
@beaulebens That was an overlook on my part. I had woo in the original mockups, I'd like to have the plugins listed by order of popularity. Woo, Akismet (usually pre-installed), WP Super Cache, VP, CrowdSignal. What do you think?

Reply:
Let's get WP Super Cache, WooCommerce, WP Job Manager, Co Authors Plus in there. Would rather not continue promoting VaultPress separately.

Reply:
@crunnells Can you work on getting the plugins listed in the order @beaulebens specified above?

Reply:
Done!